doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1099,6 +1099,7 @@ Plotting
 - Bug in :meth:`DataFrame.boxplot` and :meth:`DataFrame.plot.boxplot` lost color attributes of ``medianprops``, ``whiskerprops``, ``capprops`` and ``medianprops`` (:issue:`30346`)
 - Bug in :meth:`DataFrame.hist` where the order of ``column`` argument was ignored (:issue:`29235`)
 - Bug in :meth:`DataFrame.plot.scatter` that when adding multiple plots with different ``cmap``, colorbars alway use the first ``cmap`` (:issue:`33389`)
+- Bug in :meth:`Dataframe.plot` was rotating xticklabels when subplots was equal to True, even if the x-axis wasn't an irregular time series (:issue:`29460`)
```

Comment:
i am very sorry to be a badass to block your PR, can you change `Dataframe` to `DataFrame`? 😅

Reply:
and also double quote `subplots`
```suggestion
- Bug in :meth:`DataFrame.plot` was rotating xticklabels when ``subplots`` was equal to True, even if the x-axis wasn't an irregular time series (:issue:`29460`)
```

Reply:
:laughing: good catch, thanks!